plugins/woocommerce/tests/e2e-pw/tests/merchant/order-edit.spec.js

```diff
@@ -42,16 +42,20 @@ test.describe( 'Edit order', () => {
 	} );
 
 	test( 'can view single order', async ( { page } ) => {
-		// go to orders page
-		await page.goto( 'wp-admin/edit.php?post_type=shop_order' );
+		if ( process.env.DISABLE_HPOS === 1 ) {
```

Comment:
The value of `process.env.DISABLE_HPOS` is `'1'` (string) and this will evaluate false, executing the incorrect branch.

```suggestion
		if ( process.env.DISABLE_HPOS === '1' ) {
```